### DISCUSSION TITLE
سرویس های مرتضی باشسیز بزودی از دسترس خارج خواهد شد

### DISCUSSION THREAD

**Discussion by sh4msi** [hiddify/Hiddify-Manager#315] [General سایر]
سلام
پست دیروز (3مارچ) مرتضی در تلگرام

-------
درود
اطلاع‌رسانی

سرور bot.sudoer.net در هفته آینده خاموش خواهد شد
این به این معنی هست که آدرس‌های زیر دیگر در دسترس نخواهند بود
http://bot.sudoer.net/best.cf.iran
http://bot.sudoer.net/best.cf.iran.all
http://bot.sudoer.net/result.cf
http://bot.sudoer.net/config.real
مجدد از تمامی عزیزانی که تلاش کردند تا این مدت این سرویس‌ها رو سرپا نگه داریم ممنونم

سپاس فراوان
مرتضی باشسیز

**Comment by ghermez** [comment]
سلام 
اینو کجا اعلام کرده؟
قرار بود سرویس بات وی پی انش خاموش بشه، که شد!!
آپدیت cfscanner رو با قدرت داره جلو میبره

**Reply by sh4msi** [reply]
پست دیروز (3مارچ) مرتضی در تلگرام

----
توسعه کد های اسکنر یه موضوع جدا از ارئه عمومی سرویس هست

**Reply by ghermez** [reply]
از دسترس خارج شد.....
